exchange-app/src/app/services/state.service.ts

```diff
@@ -1,63 +1,61 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
-import exchangerate from '../interfaces/exchangerate';
-import {currencyState, valueState} from '../interfaces/state';
+import Exchangerate from '../interfaces/Exchangerate';
+import { CurrencyState, ValueState } from '../interfaces/State';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class StateService {
-  private currencyState = new BehaviorSubject<currencyState>({}) 
-  private valueState = new BehaviorSubject<valueState>({})
-  private exchangeRateState = new BehaviorSubject<exchangerate>({})
-  
-  getCurrencyState(): Observable<currencyState>{
-    return this.currencyState.asObservable()
+  private currencyState = new BehaviorSubject<CurrencyState>({});
+  private valueState = new BehaviorSubject<ValueState>({});
+  private exchangeRateState = new BehaviorSubject<Exchangerate>({});
+
+  getCurrencyState(): Observable<CurrencyState> {
```

Comment:
უფრო მოკლე და მარტივი იქნება, ეს თუ property იქნება
`public currencyState$: Observable<CurrencyState> = this.currencyState.asObservable();`

Reply:
ანუ გავაერთიანო ეს ორი ?